openapi/crmaas/crmass.json

```diff
@@ -0,0 +1,243 @@
+            "type": "string"
+          },
+          "name": "resourceTypeCode",
+          "in": "path",
+          "required": true,
+          "description": "The resourceTypeCode, here Contact"
+        },
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "fieldId",
+          "in": "path",
+          "required": true,
+          "description": "Unique identifier for field"
+        }
+      ],
+      "get": {
+        "summary": "Get field options",
+        "tags": [],
+        "responses": {
+          "200": {
+            "description": "Schema found",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "x-examples": {
+                    "Example 1": {
+                      "options": [
+                        "CRM"
+                      ]
+                    }
+                  },
+                  "properties": {
+                    "options": {
+                      "type": "array",
+                      "items": {
+                        "type": "string"
+                      }
+                    }
+                  }
+                },
+                "examples": {
+                  "Get field options": {
+                    "value": {
+                      "options": [
+                        "option1",
+                        "option2"
+                      ]
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request"
+          }
+        },
+        "operationId": "post-field-options",
+        "description": "Retrieve the options based on the given fieldId.",
+        "parameters": [
+          {
+            "schema": {
+              "type": "string"
+            },
+            "in": "query",
+            "name": "filter",
+            "description": "filter based on fieldId"
+          }
+        ]
+      }
+    }
+  },
+  "components": {
+    "schemas": {},
+    "securitySchemes": {
+      "bearerAuth": {
+        "type": "http",
+        "scheme": "bearer"
+      },
+      "OAuth2Demo": {
+        "type": "oauth2",
+        "flows": {
+          "authorizationCode": {
+            "authorizationUrl": "https://sso-api-demo.apigateway.co/oauth2/auth",
+            "tokenUrl": "https://sso-api-demo.apigateway.co/oauth2/token",
+            "scopes": {
+              "user.admin": "Read-write access to manage all users"
+            }
+          }
+        }
+      },
+      "OAuth2Prod": {
+        "type": "oauth2",
+        "flows": {
+          "authorizationCode": {
+            "authorizationUrl": "https://sso-api-prod.apigateway.co/oauth2/auth",
+            "tokenUrl": "https://sso-api-prod.apigateway.co/oauth2/token",
+            "refreshUrl": "",
+            "scopes": {
+              "user.admin": "Read-write access to manage all users"
```

Comment:
You can copy the description from https://github.com/vendasta/vendastaapis/blob/master/scopes.yaml#L54C1-L56

Reply:
In developer center `Allows read-write access to customer details` is given as description. Whereas, in the above link `Allows the application to manage the customers for the business location(s) that you manage`.
Both seems correct but Which one seems more accurate to you?

Reply:
The one that I linked to is what users see when they go through the SSO flow. A shorter one for developers is ok with me.